frontend/lib/google-analytics.tsx

```diff
@@ -188,30 +189,24 @@ export function trackPageView(pathname: string) {
  */
 const OUTBOUND_LINK_TIMEOUT_MS = 1000;
 
-/** Follow the given link. */
-function follow(href: string, target: string) {
-  if (target) {
-    window.open(href, target);
-  } else {
-    hardRedirect(href);
-  }
-}
-
 /** An event handler executed when a user clicks on an outbound link. */
 export function handleOutboundLinkClick(e: MouseEvent<HTMLAnchorElement>) {
   // If any modifier key is pressed, odds are that the user is trying to
   // invoke some browser-specific behavior to e.g. open the link in a
   // new window or tab. We don't want to break that.
   const isModifierPressed = e.altKey || e.ctrlKey || e.metaKey || e.shiftKey;
+  const { href, target } = e.currentTarget;
 
-  if (!isModifierPressed) {
-    const { href, target } = e.currentTarget;
+  const willOpenInNewWindow = target && target !== window.name;
```

Comment:
I've verified in Chrome that if our current window was opened via `target="_blank"`, its `window.name` is just the empty string, _not_ `"_blank"`, so this logic _should_ work, um, unless different browsers have different behaviors here.